app/indexers/audio_indexer.rb

```diff
@@ -1,18 +1,8 @@
 # frozen_string_literal:true
 
-class AudioIndexer < Hyrax::WorkIndexer
-  # This indexes the default metadata. You can remove it if you want to
-  # provide your own metadata and indexing.
-  include Hyrax::IndexesBasicMetadata
-
-  # Fetch remote labels for based_near. You can remove this if you don't want
-  # this behavior
-  include Hyrax::IndexesLinkedMetadata
-
-  # Uncomment this block if you want to add custom indexing behavior:
-  # def generate_solr_document
-  #  super.tap do |solr_doc|
-  #    solr_doc['my_custom_field_ssim'] = object.my_custom_property
-  #  end
-  # end
+class AudioIndexer < GenericIndexer
+  def generate_solr_document
+    super.tap do |solr_doc|
```

Comment:
Doesn't metadata still exist from generic that needs to be generated in solr here?

Reply:
Yeah, when it calls super, it uses the GenericIndexer code and generates all that.